docs/26_git_lfs.md

```diff
@@ -0,0 +1,140 @@
+
+> **Note:** The following activities utilize a the [githubschool/lfs-example](https://github.com/githubschool/lfs-example) repository, to follow along with the activities, you will need to Fork the repository to your account.
+
+### Background story
+
+ACME Incorporated is planning on using Toggl for time tracking and are using this repository to provide information on how to use Toggl and download the desktop application that has passed the IT Security team's audit.
+
+Unfortunately, the `.dmg` file was added to the Git repository without using Git LFS and we need to fix that mistake. Perform the steps found in the upcoming sections to successfully retain the `.dmg` file in the project but utilizing Git LFS to maintain it in the project.
+
+### Activity: Install Git LFS
+
+Before we can begin using Git LFS, we need to install it on our local machine. Below are the steps for installing Git LFS on your machine based on your operating system.
+
+#### MacOS
+
+1. Navigate to [git-lfs.github.com](https://git-lfs.github.com/) and click **Download**. Alternatively, you can install Git LFS using a package manager:
+
+    - [Homebrew](https://brew.sh/): `brew install git-lfs`
+
+    - [MacPorts](https://www.macports.org/): `port install git-lfs`
+
+    ?> If you install Git LFS with Homebrew or MacPorts, skip to step six.
+
+1. On your computer, locate and unzip the downloaded file.
+1. Open Terminal.
+1. Change the current working directory into the folder you downloaded and unzipped.
+
+    ```sh
+    cd ~/Downloads/git-lfs-1.X.X
+    ```
+
+    > **Note:** The file path you use after `cd` depends on your operating system, Git LFS version you downloaded, and where you saved the Git LFS download.
+
+1. To install the file, run this command:
+
+    ```sh
+    ./install.sh
+    ```
+
+    > **Note:** You may have to use `sudo ./install.sh` to install the file.
+
+1. Verify that the installation was successful:
+
+    ```sh
+    git lfs install
+    ```
+
+#### Windows
+
+1. Navigate to [git-lfs.github.com](https://git-lfs.github.com/) and click **Download**.
+
+   ?> For more information about alternative ways to install Git LFS for Windows, see this [Getting started guide](https://github.com/github/git-lfs#getting-started).
+
+1. On your computer, locate the downloaded file.
+1. Double click on the file called _git-lfs-windows-1.X.X.exe_, where 1.X.X is replaced with the Git LFS version you downloaded. When you open this file Windows will run a setup wizard to install Git LFS.
+
+    ?> When using Git LFS on windows, ensure you have version 2.29.2.windows.2 of Git or later.
```

Comment:
```suggestion
    ?> When using Git LFS on Windows, ensure you have Git version `2.29.2.windows.2` or later.
```

Reply:
Will the version always look like `2.29.2.windows.2` or should we simply say version `2.29.2` or later?

Reply:
the fix is specific to the Windows version which includes the word `windows` in the version numbering, good catch on the suggestion though ❤️